ceph.spec.in

```diff
@@ -1227,8 +1227,11 @@ export CXXFLAGS=$(echo $RPM_OPT_FLAGS | sed -e 's/-Wp,-D_FORTIFY_SOURCE=2//g')
 
 env | sort
 
-mkdir build
```

Comment:
would we also need `BuildRequires: ninja-build` dependency?

Reply:
please check the discussion we had. Nathan does not feel comfortable with building with ninja so i dropped the change.

Reply:
Yeah, I'd rather not introduce ninja into the spec file at this time. I tried it in OBS and it seemed to do more harm than good.

Reply:
how about making cmake default and ninja optional, something of sort:
```
$ ./do_cmake.sh --ninja
```

Reply:
i don't see the immediate needs for this option.